PurchasesCoreSwift/Misc/SystemInfo.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  SystemInfo.swift
+//  PurchasesCoreSwift
+//
+//  Created by Joshua Liebowitz on 6/29/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+import UIKit
+#elseif os(watchOS)
+import UIKit
+import WatchKit
+#elseif os(macOS)
+import AppKit
+#endif
+
+public typealias RCSystemInfo = SystemInfo
+
+@objc(RCSystemInfo) open class SystemInfo: NSObject {
+
+    #if targetEnvironment(macCatalyst)
+    static let PLATFORM_HEADER = "uikitformac"
+    #elseif os(iOS)
+    static let PLATFORM_HEADER = "iOS"
+    #elseif os(watchOS)
+    static let PLATFORM_HEADER = "watchOS"
+    #elseif os(tvOS)
+    static let PLATFORM_HEADER = "tvOS"
+    #elseif os(macOS)
+    static let PLATFORM_HEADER = "macOS"
+    #endif
+
+    public enum SystemInfoError: Error {
+        case invalidInitializationData
+    }
+
+    private static let defaultServerHostName = "https://api.revenuecat.com"
+
+    @objc public var finishTransactions: Bool
+    @objc public let platformFlavor: String
+    @objc public let platformFlavorVersion: String?
+    @objc public static var forceUniversalAppStore: Bool = false
+    @objc public static var isSandbox: Bool {
+        let url = Bundle.main.appStoreReceiptURL
+        guard let url = url else {
+            return false
+        }
+
+        let receiptURLString = url.path
+        return receiptURLString.contains("sandboxReceipt")
+    }
+    @objc public static func frameworkVersion() -> String { // TODO: automate the setting of this, if it hasn't been.
```

Comment:
Has it been automated?

Reply:
it gets bumped through `fastlane bump`  and `fastlane bump_and_update_changelog` https://github.com/RevenueCat/purchases-ios/blob/main/fastlane/Fastfile#L50

Reply:
updating the file path there should suffice

Reply:
Awesome.

Reply:
Updated in latest commit